### DISCUSSION TITLE
Regarding UQpy

### DISCUSSION THREAD

**Discussion by jaisinghverma** [community/community#58639] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

While running this :   
from UQpy.Distributions import Normal

I get the following error: 
ModuleNotFoundError                       Traceback (most recent call last)
Input In [3], in <cell line: 1>()
----> 1 from UQpy.Distributions import Normal

ModuleNotFoundError: No module named 'UQpy.Distributions'

**Comment by airtower-luna** [comment]
The error means that the module you're trying to import isn't available. Usually that either means it's not installed (or maybe you haven't activated a venv it's installed in?), or something is spelled wrong in the import (mind it's case sensitive).

**Comment by jaisinghverma** [comment]
How can I rectify that? Could you please explain me further?


Thanks & Regards,
Jaisingh Verma
Research Scholar
Dept. of Earthquake Engineering
IIT Roorkee
Phone: 8707653637
E-mail: ***@***.***

**Comment by airtower-luna** [comment]
Double check your spelling, and if it's correct make sure the module is installed.

If you need installation instructions check the UQpy documentation. This is a general discussion forum, I don't know the details of that.

**Comment by AlbinXXX** [comment]
It seems that the module '**UQpy**' is not installed or not properly imported. Please make sure that you have installed the '**UQpy**' module correctly. You can do this by using a package manager like pip. Open your terminal or command prompt and run the following command:

`pip install UQpy`

If the installation is successful, try importing the 'Normal' class from the 'Distributions' module again:

```
python
from UQpy.Distributions import Normal
```
If you still encounter the same error, double-check that the module is correctly installed and verify your import statements. Also, ensure that you have the latest version of the module installed, as it's possible that the module structure or import statements have changed in newer versions.

**Comment by jaisinghverma** [comment]
I checked. Everything is fine, but I am getting the error.


On Wed, Jun 21, 2023 at 6:46 PM Lorina Rushiti ***@***.***>
wrote:

> It seems that the module '*UQpy*' is not installed or not properly
> imported. Please make sure that you have installed the '*UQpy*' module
> correctly. You can do this by using a package manager like pip. Open your
> terminal or command prompt and run the following command:
>
> pip install UQpy
>
> If the installation is successful, try importing the 'Normal' class from
> the 'Distributions' module again:
>
> python
> from UQpy.Distributions import Normal
>
> If you still encounter the same error, double-check that the module is
> correctly installed and verify your import statements. Also, ensure that
> you have the latest version of the module installed, as it's possible that
> the module structure or import statements have changed in newer versions.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/58639#discussioncomment-6241882>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZY6RAOHZAWZW4QEF6CDCYLXMLX4JANCNFSM6AAAAAAZOVG77A>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Thanks & Regards,

**Reply by AlbinXXX** [reply]
Could you share the code with me so that I may take a closer look?
